solutions/security/get-started/configure-advanced-settings.md

```diff
@@ -181,6 +181,13 @@ Each time a detection rule runs using a remote cross-cluster search (CCS) index
 
 If you’ve ensured that your detection rules have the required privileges across your remote indices, you can use the `securitySolution:enableCcsWarning` setting to disable this warning and reduce noise.
 
+## Configure alert suppression window behavior [suppression-window-behavior]
+
+```yaml {applies_to}
+stack: ga 9.2
+```
+
+To control whether alert suppression continues after a suppression alert is closed during an [active suppression window](/solutions/security/detect-and-alert/suppress-detection-alerts.md#security-alert-suppression-impact-close-alerts), configure the `securitySolution:suppressionBehaviorOnAlertClosure` advanced setting. You can choose whether suppression continues or restarts when the next qualifying event meets the suppression criteria. By default, **Restart suppression** is selected.
 
```

Comment:
Should use "next qualifying alert"

Reply:
Changed in [c0dc41b](https://github.com/elastic/docs-content/pull/2735/commits/c0dc41b20d19554d0826743649a83d2387c1b4c8).